spec/integration/sec_tester_spec.cr

```diff
@@ -0,0 +1,23 @@
+require "../spec_helper"
+
+{% if env("RUN_SEC_TESTER_SPECS") == "1" %}
```

Comment:
Anyone that wants to run this spec locally would need a Bright API key... That's not really going to happen, so we can treat this like the Heroku specs. As long as this spec passes in CI, and the rest pass locally, we should be fine